clients/elrond/dataGetter.go

```diff
@@ -79,12 +79,10 @@ func (dg *elrondClientDataGetter) ExecuteQueryReturningBytes(ctx context.Context
 
 	response, err := dg.proxy.ExecuteVMQuery(ctx, request)
 	if err != nil {
-		dg.log.Error("while query SC", "request", request, "error", err)
```

Comment:
please revert. We have queries returning bytes also that won't be logged otherwise

Reply:
reverted